addons/calendar/static/src/js/calendar_renderer.js

```diff
@@ -92,15 +92,18 @@ const AttendeeCalendarPopover = CalendarPopover.extend({
             self.event.extendedProps.record.attendee_status = selectedStatus;  // FIXEME: Maybe we have to reload view
             self.$('.o-calendar-attendee-status-text').text(self.statusInfo[selectedStatus].text);
             self.$('.o-calendar-attendee-status-icon').removeClass(_.values(self.statusColors).join(' ')).addClass(self.statusInfo[selectedStatus].color);
+            self.$el.parent().hide();
```

Comment:
In commit2 ( improve the quality of the calendar app) there are changes in diff that are not really explained in commit message... not sure what they target ^^